main.go

```diff
@@ -158,8 +186,43 @@ func resourceManifestRead(d *schema.ResourceData, m interface{}) error {
 	if err := run(cmd); err != nil {
 		return err
 	}
-	if strings.TrimSpace(stdout.String()) == "" {
+	output := strings.TrimSpace(stdout.String())
+	if output == "" {
 		d.SetId("")
+		return nil
+	}
```

Comment:
The idea here is I grab the contents of the annotation `kubectl.kubernetes.io/last-applied-configuration`, normalize it, and read the content in using that. It should be correct in how it works. This allows for both terraform refresh and terraform import.